pkg/oc/cli/cmd/startbuild.go

```diff
@@ -309,6 +310,38 @@ func (o *StartBuildOptions) Complete(f *clientcmd.Factory, in io.Reader, out, er
 	return nil
 }
 
+// Validate returns validation errors regarding start-build
+func (o *StartBuildOptions) Validate() error {
+	fromCount := 0
```

Comment:
What's this variable is used for?

Reply:
It looks like you've copied it from `Complete` but it's not used here, so please remove all the code that's not necessary for `Validate`.

Reply:
Additionally, it looks like a few more checks from `Complete` deserves its places in `Validate`.